packages/server/lib/controllers/files.js

```diff
@@ -26,9 +26,15 @@ module.exports = {
 
       debug('all files to send %o', _.map(allFilesToSend, 'relative'))
 
+      const documentSuperDomainIfExists = cors.shouldInjectDocumentDomain(req.proxiedUrl, {
```

Comment:
Can we align this throughout as `shouldInjectDocumentDomain`. I think this is clearer than `documentSuperDomainIfExists`
```suggestion
      const shouldInjectDocumentDomain = cors.shouldInjectDocumentDomain(req.proxiedUrl, {
```

Reply:
The thing is this value is actually the domain value if it exists. the `shouldInjectDocumentDomain` is just the gate for it. `documentSuperDomainIfExists` in this case is either going to be the super domain string, or undefined. `shouldInjectDocumentDomain` makes the variable read like a boolean when it isn't

Reply:
I def read this as a boolean because of the `ifExists`. maybe `superDomain` is sufficent and if it has a value in the template it'll read like
```
if(superdomain) {
  document.domain = superdomain
}
```

Reply:
awesome! Addressed in 9b6938f5be